convert_color_scheme.py

```diff
@@ -0,0 +1,135 @@
+"""Functionality for converting a color scheme to a "TerminalView" scheme."""
+import os
+import plistlib
+import sublime
+
+
+def hex_to_rgb(hexstring):
+    """Convert a string representing a hex color to an RGB tuple."""
+    return (int(hexstring[1:3], 16) / 255,
+            int(hexstring[3:5], 16) / 255,
+            int(hexstring[5:7], 16) / 255)
+
+
+def rgb_to_hex(rgb):
+    """Convert an RGB tuple to a hex string."""
+    return "#{:02x}{:02x}{:02x}".format(int(rgb[0] * 255),
+                                        int(rgb[1] * 255),
+                                        int(rgb[2] * 255))
+
+
+def norm2(vec3):
+    """Compute the squared norm of a three-dimensional vector."""
+    return vec3[0]**2 + vec3[1]**2 + vec3[2]**2
+
+
+def distance2(a, b):
+    """Compute the squared distance between two 3D vectors."""
+    return norm2((a[0] - b[0], a[1] - b[1], a[2] - b[2]))
```

Comment:
I give up, I just change the yellow of my color scheme from
<img width="81" alt="screen shot 2017-08-13 at 2 28 29 pm" src="https://user-images.githubusercontent.com/1690993/29252374-b5424152-8033-11e7-8ab6-7349fbc2ee5a.png">
to
<img width="70" alt="screen shot 2017-08-13 at 4 23 18 pm" src="https://user-images.githubusercontent.com/1690993/29253133-bd443b02-8043-11e7-9a6d-43b486766243.png">
and now it works a bit better.
![Uploading Screen Shot 2017-08-13 at 4.24.08 PM.png…]()